sass/product-buys-sale/product-buys-sale.sass

```diff
@@ -0,0 +1,107 @@
+@import ./vars_product-buys-sale.sass
+
+@import ../vars.sass
+
+@import ./mixins_product-buys-sale.sass
```

Comment:
Mấy biến variables như biến màu, biến font-size, font-family, breakpoints.... để trong file _variables.scss
Mixin như flex, position,... để trong _mixin.scss
Rồi `@import` (hoặc `@use`) ở file style.sass để những file khác sử dụng chung nhé
Lưu ý phải có underscore ngay trước tên file nhé
https://stackoverflow.com/questions/34889962/why-put-in-front-of-the-file-name-or-in-scss-css